src/diffusers/loaders.py

```diff
@@ -2147,7 +2149,7 @@ def unfuse_lora(self, unfuse_unet: bool = True, unfuse_text_encoder: bool = True
             self.unet.unfuse_lora()
 
         if self.use_peft_backend:
-            from peft.tuners.tuner_utils import BaseTunerLayer
+            from peft.tuners.tuners_utils import BaseTunerLayer
```

Comment:
cc @younesbelkada

Reply:
Thanks ! I think if you merge with main branch everything should be fixed